### ISSUE TITLE
Get in touch links not descriptive enough

### ISSUE THREAD

**Issue by NickColley** [alphagov/govuk-design-system#898]
This issue is from a May 2019 external accessibility audit report.

**WCAG Reference**:
- [2.4.4 Link Purpose (In Context) (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context.html)
- [2.4.9 Link Purpose (Link Only) (Level AAA)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)

**Issue ID**: DAC_Issue5
**URL**: https://design-system.service.gov.uk/community/design-system-working-group/

## Screen Shot
![](https://user-images.githubusercontent.com/2445413/58794278-bc0a9980-85ef-11e9-9a34-582d8a105573.png)
![](https://user-images.githubusercontent.com/2445413/58794281-be6cf380-85ef-11e9-9623-e6c4562fb404.png)

There were links encountered that were not descriptive to some users, especially those that rely on audio feedback. Some users may not realise that they will be taken to the ‘Slack’ website as a result of selecting the link, especially when navigating out of the context of the page.

## Current Code Ref(s)
```html
<a class="govuk-link app-contact-panel__link" href="https://ukgovernmentdigital.slack.com/messages/govuk-design-system" data-hsupport="slack">#govuk-design-system</a>
```

## Screen reader comments
> “Below the ‘Get in touch’ heading I located a link which contained a hash tag. It might not be clear to users navigating out of context what the link relates to. Providing a clearly worded description within the link text would enable users to easily understand what will be selected. This occurred using JAWS, NVDA and VoiceOver for iPhone and Mac.”

## Cognitive user comments
> “I wasn’t sure of some of the links at the bottom of the page and what they meant.”

## Solution
Ensure that links are descriptive for users that may not understand what they are referring to i.e. ‘Slack’. More context would benefit all users that may not be familiar with the application.

**Comment by nickcolley** [comment]
Confirmed as fixed in a re-test by DAC on 7th October 2019
